data/codelists/codelist-59.yml

```diff
@@ -6,7 +6,16 @@
   '03': ReducedPriceApplicableWhenTheItemIsPurchasedAsPartOfASetOrSeriesOrCollection
   '04': VoucherPrice
   '05': ConsumerPrice
-  '06': CorporatePrice
+  '06': CorporatePrice # Actually CorporateLibraryEducationPrice but keeping CorporatePrice for backwards compatibility
```

Comment:
Do we have a way to search for this case in our ONIX to see if it is really used? We could reprocess those to allow us to change this value. Would it be working?

Reply:
I guess we can run an elasticsearch query to find matching documents, though I am not sure if we need to reprocess those books.
What is the intent behind this operation? Are we worried that those prices should not be used in retail as well? Because if that is the case I need to take this qualifier into account in the integration process fix as well.
Actually, should we have an allowlist or blocklist of qualifier for retail products?

Reply:
In fact I don't see why we should not update this to `CorporateLibraryEducationPrice`. It would only change what we export in our own ONIX files for the involved products, but I think it would in fact be a good thing? It would be up-to-date with [what the ONIX spec currently states](https://onix-codelists.io/codelist/59).